src/main/java/leets/leets_mate/LeetsMateApplication.java

```diff
@@ -5,37 +5,116 @@
 public class LeetsMateApplication {
 
     // 동작 함수입니다.
-    public void run() {
+    public void run() throws Exception {
+        Scanner sc = new Scanner(System.in);
+
+        System.out.println("[Leets 오늘의 짝에게]를 시작합니다.\n");
+
+        System.out.println("참석자들의 이름을 입력해 주세요. (,로 구분)");
+        String members = sc.nextLine();
+
+        checkHasNoEnglish(members);
+
+        System.out.println("\n최대 짝 수를 입력해 주세요.");
+        int maximumGroupSize = sc.nextInt();
+
+        List<String> memberList = parseMembers(members);
+
+        checkDataValidity(memberNumber(memberList), maximumGroupSize);
+
+        printResult(generateRandomGroups(memberList, maximumGroupSize));
+
+        while (true) {
+            System.out.print("다시 구성하시겠습니까? (y or n): ");
+            String answer = sc.next();
+
+            if (answer.equals("y")) {
```

Comment:
만약 answer이 null 이라면 어떤 문제가 있을까요?

Reply:
null일 경우 종료되는 문제가 발생하네요,,
y, n 그리고 이 두 값과 다른 경우를 예외처리 해야겠습니다 !
감사해요 !

Reply:
y 또는 n 인경우 예외처리도 중요합니다👍
그런데 answer이 null 이라면 NPE가 날 것 같아요. 

```suggestion
            if ("y".equals(answer)) {
```
이렇게 변경해도 NPE가 발생할 가능성이 있을까요?

Reply:
euqals 메서드는 nullable한 객체를 인자로 받기 때문에 NPE가 발생하지 않을 것 같은데 어떻게 생각하시는지 궁금합니다 !

Reply:
> euqals 메서드는 nullable한 객체를 인자로 받기 때문에 NPE가 발생하지 않을 것 같은데

넵 맞아요!
`answer.equals("y")`는 NPE가 발생할 가능성이 있지만 순서를 바꿔서 `"y".equals(answer)`로 작성한다면 NPE가 발생하지 않겠죠! 그래서 전 후자를 선호합니다 😊

Reply:
전혀 생각해본적 없었는데 깨닫고 갑니다 !
감사해요 !